openfisca_us/parameters/gov/states/md/tax/income/married.yaml

```
@@ -0,0 +1,46 @@
+description: MD State income tax rate for single filers
```

Please also name the file `joint_head_widow.yaml`
```suggestion
description: MD State income tax rate for joint, head of household, and widow(er) filers
```